x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlPlugin.java

```diff
@@ -344,6 +345,8 @@ public List<NamedWriteableRegistry.Entry> getNamedWriteables() {
         entries.add(EnrichLookupOperator.Status.ENTRY);
         entries.add(LookupFromIndexOperator.Status.ENTRY);
         entries.add(SampleOperator.Status.ENTRY);
+        entries.add(LinearScoreEvalOperator.Status.ENTRY);
+
```

Comment:
Don't we need to add the `RrfScoreEvalOperator` as well?

Reply:
We don't because RrfScoreEvalOperator  extends AbstractPageMappingOperator which already has its own operator.